### PR TITLE
Added ClockyClock to Js-Projects

### LINKED ISSUE #2208: [GSSoC'23] Add a Clock Website(Analog Clock and Alarm Clock)
**Describe the project you want to add with tech stack**
A fully responsive website which will contain both :
1) Analog clock                    
2) An Alarm clock
Tech Stack: HTML , CSS , JavaScript

**Expected behavior**
Users will see a homepage with options to watch the current time in analog clock as well as the option to set up the alarm. This project makes use of videos , audios, event listeners, media queries , etc to make it more interactive.

**Screenshots (optional)**
alarmClock

analogClock

### PR DESCRIPTION
**Pull request by prernasngh**
[GSSoC'23] Add a Clock Website(Analog Clock and Alarm Clock) #2208

Close #2208 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by prernasngh** [comment]
@pranjay-poddar @MohitGupta121 @Saurabh-kumar09 Sir, please review this PR made under GSSoC'23.

File: Js-Projects/ClockyClock/akram-huseyn-f3nGngsnp3A-unsplash.jpg
Comment (MohitGupta121) [inline Js-Projects/ClockyClock/akram-huseyn-f3nGngsnp3A-unsplash.jpg:1]: @prernasngh remove all images from PR and use them through link in your project. Upload it somewhere in the cloud and fetch from there.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by prernasngh** [comment]
@pranjay-poddar @MohitGupta121 @Saurabh-kumar09  Sir, I have made the required changes. Can you please review the PR under GSSoC'23 ?

File: Js-Projects/ClockyClock/pexels-mikhail-nilov-6981406.mp4
Comment (MohitGupta121) [inline Js-Projects/ClockyClock/pexels-mikhail-nilov-6981406.mp4:1]: @prernasngh please remove this video from PR, if that using in the project then upload it somewhere and then use that link to load the video

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

File: Js-Projects/ClockyClock/pexels-mikhail-nilov-6981406.mp4
Comment (prernasngh) [reply Js-Projects/ClockyClock/pexels-mikhail-nilov-6981406.mp4:1]: @MohitGupta121 Sir, I have removed the video from PR and added its link.

**Comment by prernasngh** [comment]
@MohitGupta121 Sir, is the PR eligible to be merged ?

**Review by MohitGupta121 — APPROVED** [review]

**Pull request merged**